client/src/model-source/diagram-server.ts

```diff
@@ -121,7 +115,34 @@ export abstract class DiagramServer extends ModelSource {
         }
     }
 
-    protected handleComputedBounds(action: ComputedBoundsAction): ICommand | void {
+    protected handleLocally(action: Action): boolean {
+        switch (action.kind) {
+            case RequestModelAction.KIND:
+                return this.handleRequestModel(action as RequestModelAction)
+            case ComputedBoundsAction.KIND:
+                if (!this.viewerOptions.needsServerLayout)
+                    return this.handleComputedBounds(action as ComputedBoundsAction)
+                break
+            case RequestBoundsCommand.KIND:
+                return false
+            case ExportSvgAction.KIND:
+                return this.handleExportSvgAction(action as ExportSvgAction)
+        }
+        return true
+    }
+
+    protected handleRequestModel(action: RequestModelAction): boolean {
+        if (action.options === undefined)
+            action.options = {}
```

Comment:
Please create a new one instead

Reply:
That doesn't work because the old instance would still be sent to the server. We would need to change the method signature.